ui/src/components/EntityTable/EntityTable.jsx

```diff
@@ -113,14 +132,25 @@ class EntityTable extends Component {
         </thead>
         <tbody className={c({'updating': isLoading})}>
           {result.results !== undefined && result.results.map(entity =>
-            <EntityTableRow key={entity.id}
-                            entity={entity}
-                            location={location}
-                            hideCollection={hideCollection}
-                            showPreview={showPreview}
-                            documentMode={documentMode}
-                            updateSelection={updateSelection}
-                            selection={selection} />
+            <React.Fragment>
+              <EntityTableRow key={entity.id}
+                              entity={entity}
+                              location={location}
+                              hideCollection={hideCollection}
+                              showPreview={showPreview}
+                              documentMode={documentMode}
+                              updateSelection={updateSelection}
+                              selection={selection} />
+              {entity.highlight !== undefined && entity.highlight.map(function(sentence, index){
+                return entity.name !== getHighlight(sentence) && <tr>
+                <td key={entity.id} colSpan="5" className={index !== entity.highlight.length - 1 ? 'highlighted-words' : ''}>
```

Comment:
`key` property must be set to parent element in iteration _(in our case it is `<tr>`)_